.discussions/mcp-server-feedback-2024-06-15.md

```diff
@@ -0,0 +1,53 @@
+# MCP-Server Feedback - June 15, 2024
+
+## Use-case #1:
+
+> Ask Cursor: using WorkflowAI, can you create a new agent that summarize a text?
+
+- [ ] Does the AI engineer encourages the use the input variables? At least in the code generated, input variables are not used.
+
+Generated the following completion code:
+```python
+completion = client.beta.chat.completions.parse(
+    model="text-summarizer/gpt-4o",  # Agent name following WorkflowAI convention
+    messages=[
+        {"role": "system", "content": system_prompt},
+        {"role": "user", "content": f"Please summarize the following text:\n\n{text}"}
+    ],
+    response_format=TextSummary,
+    temperature=0.3  # Slightly creative but consistent
+)
+```
+
+- [ ] The `base_url` was set to `api.workflowai.com` instead of `run.workflowai.com`. We need more context on when to use `api.workflowai.com` and when to use `run.workflowai.com`.
+
+- [ ] agent prefix needs to be added to the `metadata` field as the preferred way to identify the agent. https://github.com/WorkflowAI/fumadocs-demo/blob/main/demo/my-app/content/docs/observability/index.mdx#identify-your-agent (probably just requires the documentation repo to be merged in `workflowai/workflowai` and up-to-date?)
+
+- [ ] how does the integration between the IDE and our playground is expected to work? I would imagine that the AI engineer should be able to return a list of "next steps" after the agent was created, and open the playground?
```

Comment:
I will work on having the AI Engineer return more:
- suggested next steps
- useful link (ex: playground link, deploy page link, etc.)